manual.asc

```diff
@@ -56,22 +56,22 @@ Create deck :: Create a new empty deck
  . Choose a name for the deck, for example "New Japanese"
  . Add cards to it following the "Add" instructions above
 
-=== Action Bar
-At the top of each screen in AnkiDroid is the Action Bar, with buttons for performing various actions. 
-The following actions are available from the action bar in the deck list:
+=== App Bar
```

Comment:
Changing heading names may break links. I'm unsure if AsciiDoctor has functionality to preserve existing links, but we should look into it.

Reply:
I just read the manual, I was unable to find a way to manually specify the anchor that a section title should convert to, I think we just change it and move on